plugins/user-settings/src/components/Settings.tsx

```diff
@@ -25,10 +26,17 @@ export const Settings = () => {
   const SidebarAvatar = () => <SignInAvatar />;
 
   return (
-    <SidebarItem
-      text={displayName}
-      to={settingsRouteRef.path}
-      icon={SidebarAvatar}
-    />
+    [
+      <SidebarItem
+        text="Settings"
+        to={settingsRouteRef.path}
+        icon={SettingsIcon}
+      />,
+      <SidebarItem
+        text={displayName}
+        to={settingsRouteRef.path}
+        icon={SidebarAvatar}
```

Comment:
Hi! We only need to keep one of these - the bottom one.

Reply:
Hmm maybe I was unclear on the requirements. Did you just want to change the user avatar to a settings icon?

I put it as an additional item above because of this comment https://github.com/spotify/backstage/issues/3027#issuecomment-714248877

Reply:
Oh I'm starting to think that I am confused about the requirements as well. 😕  I thought the UI was still the popup instead of the separate settings page, but that had already been mentioned in the ticket.

So yeah, in the end, I think the desired change is to replace the name and icon, with instead "Settings" and a settings icon.

Reply:
Sorry for the confusion :)

Reply:
No problem! Will do that now.

Reply:
Should be all set.